### ISSUE TITLE
Сообщение для Эдуарда. 

### ISSUE THREAD

**Issue by SergeiAvchinnikov** [EduardFazliev/Inflo#3]
Эдуард, добрый день. Прошу связаться со мной ( можно тут или через фейсбук) . Сергей.

**Comment by EduardFazliev** [comment]
Здравствуйте, мы с вами на интервью общались?

пн, 27 апр. 2020 г., 11:41 SergeiAvchinnikov <notifications@github.com>:

**Comment by SergeiAvchinnikov** [comment]
Ага. Понравились ваши репозитории, хотелось бы вопросы задать:)

**Comment by EduardFazliev** [comment]
Конечно. Это все старые, даже немного стыдно за них, но в свое время все
работало

пн, 27 апр. 2020 г., 11:46 SergeiAvchinnikov <notifications@github.com>:

**Comment by SergeiAvchinnikov** [comment]
Девять шесть семь два шесть два шесть пять три четыре
